src/router/index.js

```diff
@@ -0,0 +1,19 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+import TxHomepage from "../components/organisms/TxHomepage.vue";
+
+Vue.use(VueRouter);
+
+const router = new VueRouter({
+  mode: "history",
+  base: import.meta.env.BASE_URL,
```

Comment:
Where is this coming from?

Reply:
sorry I forgot to remove the code above.
I will fix it